### PR TITLE
{CI} Test fix code style

### PR DESCRIPTION
**Pull request by haroldrandom**
**Description**  

The test result of https://github.com/Azure/azure-cli/pull/14402/

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -820,6 +820,8 @@ jobs:
   - bash: |
       set -ev
       . env/bin/activate
+
+      pip install --force -q git+https://github.com/Azure/azure-cli-dev-tools.git@master
```
Comment (haroldrandom) [inline azure-pipelines.yml:824]: This is the different between this PR and https://github.com/Azure/azure-cli/pull/14402/

**Comment by yonzhan** [comment]
CI

File: azure-pipelines.yml (hunk 1 above)
Comment (jiasli) [inline azure-pipelines.yml:824]: Aren't we using the PyPI version of `azdev`?

Comment (haroldrandom) [reply azure-pipelines.yml:824]: Yeah. This PR just for verifying the result, the feature is still in the master branch of azdev. I will release a newer azdev once https://github.com/Azure/azure-cli/pull/14402 is megred.

**Comment by haroldrandom** [comment]
Close as the related one is merged

**Pull request closed without merging**